ts/image-occlusion/index.ts

```diff
@@ -25,6 +30,16 @@ export async function setupImageOcclusion(mode: IOMode, target = document.body):
     checkNightMode();
     await i18n;
 
+    async function addNote(): Promise<void> {
+        addOrUpdateNote(mode, get(hideAllGuessOne));
+    }
+
+    // for adding note from mobile devices
+    globalThis.anki.imageOcclusion = {
+        mode,
+        addNote,
+    };
```

Comment:
For mobile device a button at top corner needs to add then this global method needs to call for adding/updating notes.